hudi-utilities/src/test/java/org/apache/hudi/utilities/TestHiveIncrementalPuller.java

```diff
@@ -18,27 +18,26 @@
 
 package org.apache.hudi.utilities;
 
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
 
 public class TestHiveIncrementalPuller {
 
   private HiveIncrementalPuller.Config config;
 
-  @Before
+  @BeforeEach
   public void setup() {
     config = new HiveIncrementalPuller.Config();
   }
 
   @Test
   public void testInitHiveIncrementalPuller() {
 
-    try {
+    assertDoesNotThrow(() -> {
       new HiveIncrementalPuller(config);
-    } catch (Exception e) {
-      Assert.fail("Unexpected exception while initing HiveIncrementalPuller, msg: " + e.getMessage());
-    }
+    }, "Unexpected exception while initing HiveIncrementalPuller.");
```

Comment:
`e.getMessage()` will be included in message printed by `assertDoesNotThrow()` hence no need to concat it manually.